CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Grafana k6 Studio
+
+Thank you for your interest in contributing to Grafana k6 Studio! We welcome all people who want to contribute in a healthy and constructive manner within our community. To help us create a safe and positive community experience for all, we require all participants to adhere to the [Code of Conduct](.github/CODE_OF_CONDUCT.md).
+
+If you want to chat with the team or the community, you can [join our community forums](https://community.grafana.com/c/grafana-k6/k6-studio/)
+
+## Report issues
+
+Before submitting a new issue, try to make sure someone hasn't already reported the problem. Look through the [existing issues](https://github.com/grafana/k6-studio/issues) for similar issues.
+
+Report a bug by submitting a [bug report](https://github.com/grafana/k6-studio/issues/new?template=bug.yaml).
+
+Follow the issue template and add additional information that will help us replicate the problem.
+
+## Contribute to code
+
+> [!NOTE]
+> Grafana k6 Studio is currently in active development. Most of the features are meant to be implemented by the team, so we may or may not accept contributions from the community.
+> Non-critical bug fixes and small features are welcome, but please check with us first to see if we are already working on it.
```

Comment:
This seems to be more strongly worded and gatekeepey~ as a message 🤔 

I would reword it as Small features & bug fixes are welcome but for bigger contributions to please discuss them in a github ticket before starting to work on it.

A potential contributor shouldn't have to ask us to see if we are working on something, that should be clear from issues and assignees, if that's not the case it means we have to improve on that 🤔

Reply:
I think these are some arguments, but I think the wording is appropriate for the current stage of the project:

> A potential contributor shouldn't have to ask us to see if we are working on something, that should be clear from issues and assignees, _if that's not the case it means we have to improve on that_

This is currently not the case, but I agree that we should improve on that and change the document accordingly.

> Small features & bug fixes are welcome but for bigger contributions to please discuss them in a github ticket before starting to work on it

This would be in line with the k6 contributing guidelines, but how do we decide what makes a feature too big to accept contributions?

Reply:
Maybe we could say something like "Contributions for bug fixes and issues tagged with `help-wanted` are welcome but for bigger contributions to please discuss them in a github ticket before starting to work on it"? That would make it a part of triaging issues.

Reply:
yeah that sounds nicer, I still think the current wording seems to strong

> This would be in line with the k6 contributing guidelines, but how do we decide what makes a feature too big to accept contributions?

We will have to decide on a per issue base, some will be easier to spot but for other might require some discussion

Reply:
> We will have to decide on a per issue base

I was asking more from the external contributor's perspective: they may lack context and not always be able to figure out whether the issue is big/small. I like the `help-wanted` label suggestion